app/public/js/hover_text.js

```diff
@@ -70,6 +71,14 @@ $(document).ready(function()
 	$location_tag_elements.focus(function(){
 		delayedHide(location_tag_descriptions[getLocationTagId($(this))]);
 	});
+	$location_tag_elements.bind('touchend',function(e){
```

Comment:
Just in case we forget the status of this pull request, we're waiting for feedback from Blandine on the issue before merging this.  It was difficult to reproduce the problem as she described and it looked like the user was confusing an intentional delay with a performance issue.